packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -371,20 +371,23 @@ class Site extends React.Component<
         super(props);
 
         this.initialPath = this.getInitialPath();
-
+        const path: string =
+            this.initialPath === this.getCurrentPath()
+                ? this.initialPath
```

Comment:
I think you can just assign this to `this.getCurrentPath()`

Reply:
Fixed it.